### PR TITLE
Add macOS left CMD and right CMD key code

### PR DESCRIPTION
**Pull request by igogo5yo**
Add macOS left CMD and right CMD key code

### CODE REVIEW

File: lib/KeyCode.js [hunk 1]
```diff
@@ -61,6 +61,7 @@ exports.KEY_W = 87;
 exports.KEY_X = 88;
 exports.KEY_Y = 89;
 exports.KEY_Z = 90;
+exports.KEY_LEFT_CMD = 91;
```
Comment (mesaugat) [inline lib/KeyCode.js:64]: Looks good to me. While you are at it, can you also add `exports.KEY_RIGHT_CMD = 93`.

It is also the context menu button in Windows keyboard layout.

Comment (mesaugat) [reply lib/KeyCode.js:64]: Can you also add tests here: https://github.com/kabirbaidhya/keycode-js/blob/master/test/KeyCode.spec.js

And also update the TypeScript definition file: https://github.com/kabirbaidhya/keycode-js/blob/master/index.d.ts

Comment (kabirbaidhya) [reply lib/KeyCode.js:64]: Additionally, you'll also need to update the README once you made these changes.

Comment (igogo5yo) [reply lib/KeyCode.js:64]: README and KeyCode.spec has been updated.

> Looks good to me. While you are at it, can you also add `exports.KEY_RIGHT_CMD = 93`.
> 
> It is also the context menu button in Windows keyboard layout.

@mesaugat 
So, you want to duplicate the 93 for two different KEY names?

Comment (mesaugat) [reply lib/KeyCode.js:64]: Yes, correct.

Comment (igogo5yo) [reply lib/KeyCode.js:64]: Done

Comment (igogo5yo) [reply lib/KeyCode.js:64]: @kabirbaidhya if you will merge this request, please make new release, I very need this changes in one of my projects.

Comment (mesaugat) [reply lib/KeyCode.js:64]: @igogo5yo Sure, we will make a release.

One last change we need is to add the constants here: https://github.com/kabirbaidhya/keycode-js/blob/master/index.d.ts

**Review by mesaugat — APPROVED** [review]
💯 Thank you for the update.

**Comment by kabirbaidhya** [comment]
Patched [v1.0.4](https://github.com/kabirbaidhya/keycode-js/releases/tag/v1.0.4).

**Pull request merged**